src/Preflight/Preflight.php

```diff
@@ -276,7 +276,9 @@ public function preflight($argv)
 
         // If we did not redispatch, then add the site-wide config for the
         // new root (if the root did in fact change) and continue.
-        $this->configLocator->addSitewideConfig($root);
+        $this->configLocator->addDrupalConfig($root);
+        // Set multisite config using uri from alias.
+        $this->configLocator->addSiteConfig($root, $selfAliasRecord);
```

Comment:
Is it necessary to pass in `$root` here? i.e. will $root ever be different than the value held by the self alias record? If it is, wouldn't the value in the self alias be more correct?